sema/ro/creator/rocreator.py

```
@@ -0,0 +1,56 @@
+        :param force: overwrites the output file even if it exists
+        :type force: bool
+        """
+        # upfront checks
+        # path resolving
+        # existance and writeability checks
+        self._result = RocResult()
+        ...
+
+    @Trace.init(Trace)
+    def process(self) -> RocResult:
+        ...
```

Ruff (0.8.2)

Unnecessary `...` literal

Remove unnecessary `...`

(PIE790)